app.py

```diff
@@ -76,44 +75,6 @@ def proposals():
     return jsonify({'proposals': service_proposals})
 
 
-# TODO: make 'session_id' param required after client update
-@app.route('/v1/client_create_session', methods=['POST'])
-@validate_json
-def client_create_session():
-    payload = request.get_json(force=True)
-    node_key = payload.get('node_key', '')
-
-    node = Node.query.get(node_key)
-    if not node:
-        return jsonify(error='node key not found'), 400
-
-    if node.get_status() != 'active':
```

Comment:
Are You listing just `active` proposals in `v1/proposals`?

Reply:
Yes - this logic is in the model itself.